namespace-provisioner/use-case7.hbs.md

```diff
@@ -0,0 +1,469 @@
+            ssh_privatekey: SSH-PRIVATE-KEY
+            identity: SSH-PRIVATE-KEY
+            identity_pub: SSH-PUBLIC-KEY
+            known_hosts: GIT-SERVER-PUBLIC-KEYS
+      EOF
+      ```
+
+2. To create a secret to be added to the service account in the developer namespace within the GitOps repository, you can use this [example](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/gitops-airgap/resources/git.yaml) for HTTP-based or this [example](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/ns-provisioner-samples/gitops-airgap/resources/settings-xml.yaml) for `setings.xml`-based, or follow the provided example below.
+
+    Rather than directly including the actual username and password in the Git repository secret,
+    utilize the `data.values.imported` keys to add references to the values from the `git-auth` secret
+    created in Step 1.
+
+    This secret represents the actual Git secret that will be created by the Namespace Provisioner
+    in each managed namespace. It should be included in your Git repository linked in the
+    `additional_sources` section of `tap-values.yaml` mentioned in Step 4.
+
+    Using HTTP(s) based Authentication
+    : If using Username and Password for authentication.
+
+      
+      Assuming that, in this configuration for an air-gapped environment,
+       the Git Repository server has a custom certificate of authority that
+       cannot be verified against public issuers, you must provide
+       the `caFile` content to log in against it.
+       
+       ```yaml
+      #@ load("@ytt:data", "data")
+      ---
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: git
+        annotations:
+          tekton.dev/git-0: #@ data.values.imported.git.host
+      type: kubernetes.io/basic-auth
+      stringData:
+        username: #@ data.values.imported.git.username
+        password: #@ data.values.imported.git.token
+        caFile: #@ data.values.imported.git.caFile
+      ```
+
+    Using settings.xml based Authentication for Java applications
+    : If using Username and Password for authentication.
+       
+       ```yaml
+      #@ load("@ytt:data", "data")
+
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: settings-xml
+      type: service.binding/maven
+      stringData:
+        type: maven
+        provider: sample
+        #@yaml/text-templated-strings
+        settings.xml: |
+          <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+              <mirrors>
+                  <mirror>
+                      <id>reposilite</id>
+                      <name>Tanzu seal Internal Repo</name>
+                      <url>https://reposilite.tap-trust.cf-app.com/releases</url>
```

Comment:
Should match template from the accelerator repo.

Reply:
Updated